pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -658,20 +665,15 @@ func (l *listUsersQuery) expandExclusion(
 	wildcardKey := tuple.TypedPublicWildcard(req.GetUserFilters()[0].GetType())
 
 	_, baseWildcardExists := baseFoundUsersMap[wildcardKey]
-	subtractWildcardUser, subtractWildcardExists := subtractFoundUsersMap[wildcardKey]
-
-	if baseWildcardExists && (!subtractWildcardExists || subtractWildcardUser.relationshipStatus == NoRelationship) {
-		trySendResult(ctx, foundUser{
-			user: tuple.StringToUserProto(wildcardKey),
-		}, foundUsersChan)
-	}
```

Comment:
**Note:** Realizing in hindsight that this is probably an easy optimization we can apply. So we'll need to re-add this back in and remove the redundant code in the main loop below.